admin_guide/idling_applications.adoc

```diff
@@ -0,0 +1,71 @@
+[[dev-guide-idling-applications]]
+= Idling Applications
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+:prewrap!:
+
+toc::[]
+
+== Overview
+
+As an {product-title} administrator, you can idle applications in order to
+reduce the consumption of resources. 
+
+Applications are made of services, as well as resources, such as deployment
```

Comment:
"as well as scalable resources" (services themselves are also a type of resource ;-) )